frontend/src/data/sorting/AgeSorterStrategy.ts

```diff
@@ -0,0 +1,58 @@
+import { Breakdowns } from "../query/Breakdowns";
+import { Row } from "../utils/DatasetTypes";
+import { AbstractSortStrategy } from "./AbstractDataSorter";
+
+export class AgeSorterStrategy extends AbstractSortStrategy {
+  frontValues: string[];
+  backValues: string[];
```

Comment:
I sort of wonder - if this is something that's going to apply across multiple sorters, can it be shared? The front/back values complicates the compare function a good bit and needs to be repeated. This could be handled by moving the front/back values to the DatasetOrganizer or to the base class, though there are some subtleties there for whether you use them inside the compare function or outside the compare function.

This isn't a dealbreaker, up to you whether you want to address this or not. Don't know if we'll really need that many sorters in the long run.